gatsby/plonerestapi.rst

```diff
@@ -0,0 +1,131 @@
+Fetching Data Using Plone REST.API
+==================================
+
+All the data from a Plone site is available in the JSON format using the `plone.restapi <https://plonerestapi.readthedocs.io/en/latest/introduction.html>`_.
+
+We'll be working a lot with this API while working on the Gatsby source-plugin, so it is recommended that you have an API browser to explore the API.
+Install `Postman <https://www.getpostman.com/>`_ and do go through the quick guide to working with plone.restapi:
+
+https://plonerestapi.readthedocs.io/en/latest/exploring.html#exploring-api-postman-onboarding
+
+.. note:: We're using the same endpoints as for loading the site on a browser but setting the header `Accept: application/json` is what retrieves JSON data for us.
```

Comment:
Put content of note directive two lines further down and indent. Do not use contractions (we're). Shorter sentences. Grammar fixes:

```
.. note::

    We will use the same endpoints for loading the site in a browser, but set the header ``Accept: application/json``.
    This header retrieves JSON data for us.
```